HOWTO.md

```diff
@@ -734,3 +736,26 @@ Options:
 
 Please note that in this mode there are some environment variables that are
 still required to be set by the user (`DI_SIGN_KEY_PATH`, `DI_HMAC_KEY_PATH`).
+
+## How to use Features
+
+### How to use `per-device serviceinfo` feature
```

Comment:
I think that a "the" is needed in "How to use the `per-device serviceinfo` feature"